src/components/data_display/routes.ts

```diff
@@ -152,24 +156,80 @@ const routeDataViewFilterList = async (request: any, reply: any) => {
 
   const dataView = dataViews[0]
 
-  // Check column exists on highest priority dataView
-  if (!dataView.tableViewIncludeColumns?.includes(columnName))
-    throw new Error(`Column "${dataViewCode}" not available on data view: "${dataView.title}"`)
+  // TO-DO: Create search filters for types other than string (number, bool, array)
+  const searchFilter =
+    searchText === ''
+      ? {}
+      : {
+          or: searchFields.map((col: string) => ({
+            [col]: {
+              includesInsensitive: searchText,
+            },
+          })),
+        }
+
+  const gqlFilters = { ...getFilters(dataView, userId, orgId), ...searchFilter }
```

Comment:
We probably only need to search "string" fields for filter lists, as other types won't be "list" type (e.g. Boolean -> valueMap, date -> Date picker, number -> range selector).

But we can extend this later if we find a use case where it's necessary to have a non-string filter list.